src/modules/Channel/components/SuggestedMentionList/index.tsx

```diff
@@ -56,7 +57,7 @@ function SuggestedMentionList(props: SuggestedMentionListProps): JSX.Element {
   const [searchString, setSearchString] = useState('');
   const [lastSearchString, setLastSearchString] = useState('');
   const [currentUser, setCurrentUser] = useState<User>(null);
```

Comment:
nit:
Can we change the variable name of **currentUser** even though it's unrelated to this PR's changes? (e.g., to currentFocusedMember)
Normally, in our business logic, **currentUser** refers to the user currently connected to the SDK, so the name can be quite confusing.

Reply:
ah thanks for finding this out. ill change to currentFocusedMember